### PR TITLE
fix: Update old packages in internal libraries

### PR DESCRIPTION
**Pull request by JhontSouth**
#minor

## Description
This PR upgrades old packages of internal libraries that don't have conflicts with BotBuilder-JS in their latest versions.

## Specific Changes
  - Updated the following packages:
      - @babel/core
      - @babel/preset-env 
      - @babel/preset-typescript 
      - @babel/runtime
      - @types/minimist 
      - @types/mocha
      - @types/request 
      - @types/restify 
      - @types/semver 
      - @types/uuid
      - assert 
      - async
      - babel-loader
      - bitwise
      - clean-webpack-plugin
      - colors
      - core-js 
      - cross-fetch 
      - dayjs
      - dotenv 
      - fs-extra 
      - mime 
      - minami
      - minimist
      - nanoid
      - nock
      - nyc
      - promise 
      - random-js
      - semver
      - should
      - sinon 
      - swagger-client 
      - ts-node
      - typescript
      - underscore
      - uuid 
      - webpack
      - xmlbuilder
      - zod

## Testing
The following images show some tests working after the updates.
image
image

### CODE REVIEW

File: testing/transcripts/package.json [hunk 1]
```diff
@@ -9,14 +9,14 @@
     "author": "Microsoft Corp.",
     "license": "MIT",
     "dependencies": {
-        "@types/node": "^10.17.27",
-        "@types/restify": "^8.4.2",
+        "@types/node": "^22.5.0",
```
Comment (ceciliaavila) [inline testing/transcripts/package.json:12]: We discussed this. We are going to use @types/node 18.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]
Left a comment about types/node.
Also, can we run an audit to make sure we are not introducing new issues?

File: testing/transcripts/package.json (hunk 1 above)
Comment (JhontSouth) [reply testing/transcripts/package.json]: fixed

**Review by ceciliaavila — APPROVED** [review]

**Comment by sw-joelmut** [comment]
moved to #471

**Pull request closed without merging**